src/components/input/test/Select-test.js

```diff
@@ -99,6 +99,45 @@ describe('Select', () => {
   const clickOption = (wrapper, id) =>
     wrapper.find(`[data-testid="option-${id}"]`).simulate('click');
 
+  function clickOptionCheckbox(wrapper, id) {
+    const checkbox = wrapper
+      .find(`[data-testid="option-${id}"]`)
+      .closest('[role="option"]')
+      .find('[type="checkbox"]');
+
+    if (!checkbox.exists()) {
+      throw new Error(
+        `There is no checkbox in option ${id}. Make sure you are testing a MultiSelect`,
+      );
+    }
+
+    checkbox.simulate('change');
+  }
+
+  const pressKeyInOption = (wrapper, id, keyCode) =>
+    wrapper
+      .find(`[data-testid="option-${id}"]`)
+      .closest('[role="option"]')
+      .getDOMNode()
+      .dispatchEvent(new KeyboardEvent('keydown', { code: keyCode }));
```

Comment:
See comments about `key` vs `code` above.

Note that you can use `simulate('keydown', { args...} )` as well here instead of `getDOMNode().dispatchEvent(...)`.